examples/nasjob-example-RL.yaml

```diff
@@ -0,0 +1,142 @@
+apiVersion: "kubeflow.org/v1alpha1"
+kind: StudyJob
+metadata:
+  namespace: kubeflow
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: nas-rl-example
+spec:
+  studyName: nas-rl-example
+  owner: crd
+  optimizationtype: maximize
+  objectivevaluename: Validation-accuracy
+  optimizationgoal: 0.99
+  requestcount: 4
+  metricsnames:
+    - accuracy
+  nasConfig:
+    graphConfig:
```

Comment:
Please explain about graphConfig and operations.
I believe The graphConfig is general information for NAS Job and Operations are task-specific info e.g. CNN.
Why not merge them? The graphConfig is able to encode to parameterconfig.

Reply:
Yes, GraphConfig is a initial general information for the Directed Acyclic Graph (DAG). We can't merge them, because for each StudyJob in NAS GraphConfig should be single and Operations can be as many as you want. Also, right now structure of Parameter config doesn't match with GraphConfig, we have to extend ParameterConfig as well in that case.